elasticsearch/helpers/__init__.py

```diff
@@ -268,7 +277,10 @@ def scan(client, query=None, scroll='5m', raise_on_error=True, preserve_order=Fa
     if not preserve_order:
         kwargs['search_type'] = 'scan'
     # initial search
-    resp = client.search(body=query, scroll=scroll, **kwargs)
+    search_kwargs = search_kwargs or {}
+    search_kwargs.update(global_kwargs)
```

Comment:
I think these need to be reversed - start with `global_kwargs.copy()` and then do `.update(search_kwargs)` - that way `search_kwargs` will have priority and will be able to override `global_kwargs`. Other way might be counter intuitive.